src/main/resources/default/assets/javascript/multiLanguageField.js

```diff
@@ -159,17 +271,33 @@ MultiLanguageField.prototype.updateLanguageManagementOptions = function () {
         return;
     }
 
-    const _addLanguageButton = this._modal.querySelector('.mls-add-language-button');
+    let _addLanguageButton;
+    if (this.multiline) {
```

Comment:
je nachdem wie viel code sich das am ende teilt würde ich das in zwei methoden aufrufen, also a la:
```
if (this.multiline) {
 this.buildMultiLineStuff();
} else {
 this.buildSingleLineStuff();